.sqlfluff

```diff
@@ -14,7 +17,7 @@ templater = jinja
 # See https://docs.sqlfluff.com/en/stable/configuration.html#enabling-and-disabling-rules
 # AM04 (ambiguous.column_count) and ST06 (structure.column_order) are
 # two of the more controversial rules included to illustrate usage.
-exclude_rules = ambiguous.column_count, structure.column_order
+exclude_rules = ambiguous.column_count, structure.column_order, layout.long_lines
```

Comment:
I'm adding a rule exemption for long lines, which I'm not happy about, but we have something in our `project_list_view` that puts `sqlfluff` into an infinite loop. We may be able to get this rule reinstated, but it felt outside the scope of this project.